src/Evolution/Systems/GeneralizedHarmonic/DampedHarmonicGaugeQuantities.cpp

```diff
@@ -0,0 +1,928 @@
+  buffer.d3_spatial_metric =
+    deriv_spatial_metric<SpatialDim, Frame, DataVector>(phi);
+
+  // commonly used terms
+  const auto exp_fac_1 = 1. / 2.;
+  const auto exp_fac_2 = 0.;
+  get(buffer.one_over_lapse) = 1. / get(lapse);
+  buffer.log_fac_1 = DampedHarmonicGaugeHelpers::log_factor<DataVector>(lapse,
+    sqrt_det_spatial_metric, exp_fac_1);
+  buffer.log_fac_2 = DampedHarmonicGaugeHelpers::log_factor<DataVector>(lapse,
+    sqrt_det_spatial_metric, exp_fac_2);
+
+  // Tempering functions
+  const double roll_on_HI =
+    DampedHarmonicGaugeHelpers::roll_on_function(time, t_start_HI, sigma_t_HI);
+  const double roll_on_L1 =
+    DampedHarmonicGaugeHelpers::roll_on_function(time, t_start_L1, sigma_t_L1);
+  const double roll_on_L2 =
+    DampedHarmonicGaugeHelpers::roll_on_function(time, t_start_L2, sigma_t_L2);
+  const double roll_on_S =
+    DampedHarmonicGaugeHelpers::roll_on_function(time, t_start_S, sigma_t_S);
+  buffer.weight = DampedHarmonicGaugeHelpers::weight_function<SpatialDim,
+    Frame, DataVector>(coords, r_max, r_width);
+
+  // coeffs that enter gauge source function
+  get(buffer.mu_L1) = amp_coef_L1 * roll_on_L1 * get(buffer.weight) *
+    pow(get(buffer.log_fac_1), exp_L1);
+  get(buffer.mu_S)  = amp_coef_S  * roll_on_S  * get(buffer.weight) *
+    pow(get(buffer.log_fac_1), exp_S);
+  get(buffer.mu_L2) = amp_coef_L2 * roll_on_L2 * get(buffer.weight) *
+    pow(get(buffer.log_fac_2), exp_L2);
+  get(buffer.mu_S_over_N) = get(buffer.mu_S) * get(buffer.one_over_lapse);
+
+  // Calc \f$ \mu_1 = \mu_{L1} log(rootg/N) = R W log(rootg/N)^5\f$
+  get(buffer.mu1) = get(buffer.mu_L1) * get(buffer.log_fac_1);
+
+  // Calc \f$ \mu_2 = \mu_{L2} log(1/N) = R W log(1/N)^5\f$
+  get(buffer.mu2) = get(buffer.mu_L2) * get(buffer.log_fac_2);
+
+  const auto d0_roll_on_HI = DampedHarmonicGaugeHelpers::
+    time_deriv_of_roll_on_function(time, t_start_HI, sigma_t_HI);
+  const auto d0_roll_on_L1 = DampedHarmonicGaugeHelpers::
+    time_deriv_of_roll_on_function(time, t_start_L1, sigma_t_L1);
+  const auto d0_roll_on_S = DampedHarmonicGaugeHelpers::
+    time_deriv_of_roll_on_function(time, t_start_S, sigma_t_S);
+  const auto d0_roll_on_L2 = DampedHarmonicGaugeHelpers::
+    time_deriv_of_roll_on_function(time, t_start_L2, sigma_t_L2);
+
+  // Calc \f$ \partial_a [R W] \f$
+  buffer.d4_weight =
+    DampedHarmonicGaugeHelpers::spacetime_deriv_of_weight_function<SpatialDim,
+      Frame, DataVector>(coords, r_max, r_width);
+  buffer.d4_RW_L1 = buffer.d4_weight;
+  buffer.d4_RW_S  = buffer.d4_weight;
+  buffer.d4_RW_L2 = buffer.d4_weight;
+  for (size_t a = 0; a < SpatialDim + 1; ++a) {
+    buffer.d4_RW_L1.get(a) *= roll_on_L1;
+    buffer.d4_RW_S.get(a)  *= roll_on_S;
+    buffer.d4_RW_L2.get(a) *= roll_on_L2;
+  }
+  get<0>(buffer.d4_RW_L1) += get(buffer.weight) * d0_roll_on_L1;
+  get<0>(buffer.d4_RW_S)  += get(buffer.weight) * d0_roll_on_S;
+  get<0>(buffer.d4_RW_L2) += get(buffer.weight) * d0_roll_on_L2;
+
+  /*!
```

Comment:
Remove `!` since this is not rendered by doxygen.

Reply:
see above [here](https://github.com/sxs-collaboration/spectre/pull/1172#discussion_r253735138)